components/json/rs/src/lib.rs

```diff
@@ -212,3 +213,32 @@ pub fn convert_uint64_in_object_array_to_string(path: &str, key: &str, json: &st
 
     serde_json::to_string(&unwrapped_value).unwrap_or_else(|_| "".into())
 }
+
+// Parses and re-serializes json with all Number values (u64/i64/f64) converted to strings.
+// Values other than u64/i64/f64 are unchanged. The fields could be arbitrarily nested, as
+// the entire conversion is applied to the entire JSON recursively.
+//
+// Returns an empty String if such conversion is not possible.
+//
+// Examples: convert_all_numbers_to_string(json)
+// {"a":1,"b":-1,"c":3.14} -> {"a":"1","b":"-1","c":"3.14"}
+// {"some":[{"deeply":{"nested":[{"path":123}]}}]} -> {"some":[{"deeply":{"nested":[{"path":"123"}]}}]}
```

Comment:
I recommend triple-slash comments (`/// ...`) on function headers, as those will automatically be recognized as doc comments